src/main/java/seedu/us/among/logic/commands/ShowCommand.java

```diff
@@ -19,9 +19,11 @@
 
     public static final String COMMAND_WORD = "show";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows the details of an existing API endpoint "
-            + "identified using its displayed index from the saved endpoint list.\n"
+    public static final String MESSAGE_SUCCESS = "Endpoint details: \n%1$s";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows the details of an existing endpoint "
+            + "using its displayed index from the endpoint list.\n"
             + "Parameters: INDEX\n"
+            + "Use the help command for more information.\n\n"
```

Comment:
'Use the help command for more information' can be refactored into MEsSSAGES. Would make changing this easier